workflow/executor/pns/pns.go

```diff
@@ -395,7 +395,7 @@ func (p *PNSExecutor) GetTerminatedContainerStatus(ctx context.Context, containe
 			containerStatus = &containerStatusRes
 			return containerStatus.State.Terminated != nil, nil
 		}
-		return false, errors.New(errors.CodeNotFound, fmt.Sprintf("containerID %q is not found in the pod %s", containerID, p.podName))
+		return false, nil
```

Comment:
I've noted that it can take a bit longer to get the container IDs in the pod status - so we should not error - we should retry

Reply:
Same as the previous comment, the original error needs to return from this function instead of `ErrWaitTimeout`